packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/integ.evaluate-expression-default-runtime.ts

```diff
@@ -0,0 +1,56 @@
+import { App, Stack, StackProps } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import { Construct } from 'constructs';
+import * as sfn from 'aws-cdk-lib/aws-stepfunctions';
+import * as cdk from 'aws-cdk-lib';
+import * as tasks from 'aws-cdk-lib/aws-stepfunctions-tasks';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --input '{"a": 3, "b": 4}' --state-machine-arn <StateMachineARN>
+ * * aws stepfunctions describe-execution --execution-arn <execution-arn>
+ * * The output here should contain `status: "SUCCEEDED"` and `output: "{ a: 3, b: 4, c: 7, d: 14, now: <current date> }"
+ */
+
+class TestStack extends Stack {
+  constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const sum = new tasks.EvaluateExpression(this, 'Sum', {
+      expression: '$.a + $.b',
+      resultPath: '$.c',
+    });
+
+    const multiply = new tasks.EvaluateExpression(this, 'Multiply', {
+      expression: '$.c * 2',
+      resultPath: '$.d',
+    });
+
+    const now = new tasks.EvaluateExpression(this, 'Now', {
+      expression: '(new Date()).toUTCString()',
+      resultPath: '$.now',
+    });
+
+    const statemachine = new sfn.StateMachine(this, 'StateMachine', {
+      definition: sum
+        .next(multiply)
+        .next(
+          new sfn.Wait(this, 'Wait', {
+            time: sfn.WaitTime.secondsPath('$.d'),
+          }),
```

Comment:
is there a reason to add a wait? Just wondering since it will make the test slower